### DISCUSSION TITLE
Add Advancement for Chunkloader

### DISCUSSION THREAD

**Discussion by Ayutac** [TechReborn/TechReborn#2951] [Ideas]
because there isn't one

**Comment by purejosh** [comment]
Not everything needs an advancement. In fact, installing too many mods can cause crashes related to too many advancements. So, this is really not a great idea.

**Reply by Ayutac** [reply]
TR has advancements for nearly everything, so this would simply fit.

Given the amount of advancements TR already has, + the even more recipe advancements I think this argument is invalid.

**Reply by purejosh** [reply]
Ah, now I understand your other post.
